styleguide/derek/view-templates/stories/_stories.scss

```diff
@@ -1,3 +1,218 @@
+// Customer Stories Feature Colors
+$cust-stories-red: #9d0000;
+$cust-stories-teal: #00b4c3;
+$cust-stories-purple: #563777;
+$default-bg-img: 'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/IMACbanners/collaboration-2.jpg';
+
+.story-featured-container {
+  background-color: $gray-lightest;
+  padding: $standard-padding;
+}
+
+.story-featured-text {
+  @include make-container;
+  text-align: center;
+}
+
+.story-featured-cards {
+  @include make-container;
+}
+
+.story-freatured-cardRow {
+  @include make-row;
+  margin-top: 10px;
+}
+
+.story-featured-cardLogo {
+  @include flex-center;
+  @include make-md-column(3);
+  box-shadow: 0 1px 3px transparentize($black, .5);
+  display: flex;
+  height: 220px;
+  padding: 10px;
+}
+
+.story-featured-cardQuote {
+  display: none;
+}
+
+.story-featured-link {
+  &:hover {
+    &::after {
+      opacity: 1;
+      transition-duration: .3s;
+    }
+  }
+}
+
+// 3 featured logo / quote hover states
+.story-featured-cardTop {
+  .story-featured-cardLogo {
+    background-color: $cust-stories-red;
+  }
+
+  .story-featured-link {
+    &:hover {
+      &::after {
+        background-image: linear-gradient(to bottom right, $cust-stories-red, $cust-stories-red);
+      }
+    }
+  }
+}
+
+.story-featured-cardCenter {
+  .story-featured-cardLogo {
+    background-color: $cust-stories-teal;
+  }
+
+  .story-featured-link {
+    &:hover {
+      &::after {
+        background-image: linear-gradient(to bottom right, $cust-stories-teal, $cust-stories-teal);
+      }
+    }
+  }
+}
+
+.story-featured-cardBottom {
+  .story-featured-cardLogo {
+    background-color: $cust-stories-purple;
+  }
+
+  .story-featured-link {
+    &:hover {
+      &::after {
+        background-image: linear-gradient(to bottom right, $cust-stories-purple, $cust-stories-purple);
+      }
+    }
+  }
+}
+
+// Desktop only rules
+@media (min-width: $screen-md-min) {
```

Comment:
Thank you for the mobile-first approach here.  Would you mind moving this code to the bottom of the file with the rest of the media queries?